tests/resources/OmniFixture-dump.sql

```diff
@@ -1267,7 +1272,7 @@ CREATE TABLE `user` (
 
 LOCK TABLES `user` WRITE;
 /*!40000 ALTER TABLE `user` DISABLE KEYS */;
-INSERT INTO `user` VALUES ('superUserLogin','$2y$10$mbMrIm8ZN82gFoNsV76Z3.avOWosOlT.D/6Pz4EZzhmxbedUeOyzm','hello@example.org','',1,'2020-04-15 14:02:31','2020-04-15 14:02:31',null);
+INSERT INTO `user` VALUES ('superUserLogin','$2y$10$mbMrIm8ZN82gFoNsV76Z3.avOWosOlT.D/6Pz4EZzhmxbedUeOyzm','hello@example.org','',1,'2020-04-15 14:02:31','2020-04-15 14:02:31','2020-04-15 14:02:31',null,null,null,null);
```

Comment:
The OmniFixture SQL doesn't need updating as it just creates the initial database state, then all the migration updates are applied which will add the columns with default values.

Reply:
@bx80 some how it was given an error test on UI test for that one.

Reply:
That could have been because of the issue with the migration addColumn() parameters. If it's working now then probably okay to leave it as is.